lib/ts/types.ts

```diff
@@ -17,25 +17,26 @@ import RecipeModule from "./recipeModule";
 import NormalisedURLDomain from "./normalisedURLDomain";
 import NormalisedURLPath from "./normalisedURLPath";
 import { TypeFramework } from "./framework/types";
+import { BaseRequest } from "./framework";
```

Comment:
Change the import for `TypeFramework` to `import type` if it is just a type import